include/cantera/zeroD/Reactor.h

```diff
@@ -186,6 +186,15 @@ class Reactor : public ReactorBase
     //! Reset the reaction rate multipliers
     virtual void resetSensitivity(double* params);
 
+    //! Return a false if preconditioning is not supported or true otherwise.
+    //!
+    //! @warning  This method is an experimental part of the %Cantera
+    //! API and may be changed or removed without notice.
+    //!
+    //! @since New in Cantera 3.0
+    //!
+    virtual const bool preconditionerSupported() {return false;};
+
```

Comment:
This should be:
```suggestion
    virtual bool preconditionerSupported() const {return false;};

```
The point is that the method doesn't modify the `Reactor`, not that the return value is immutable, which is what this currently says. Ditto on all of the other implementations of this method.